python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/management-security.j2

```diff
@@ -86,6 +86,7 @@ management security
 {%         endfor %}
 {%     endfor %}
 {%     for ssl_profile in management_security.ssl_profiles | arista.avd.natural_sort %}
+   !
    ssl profile {{ ssl_profile.name }}
 {%         if ssl_profile.tls_versions is arista.avd.defined %}
       tls versions {{ ssl_profile.tls_versions }}
```

Comment:
trust certificate system is not in correct order

Reply:
also many other config is not in correct order for ssl profile, please check

Reply:
It is sorting trust_certificate.certificates. trust_certificate.system is outside of this, so there is config mismatch.

Reply:
No, this wont fix the order.